package.json

```
@@ -12,11 +12,12 @@
     "node": "20.x",
     "yarn": ">=1.22.19"
   },
+  "type": "module",
   "scripts": {
-    "start": "craco start",
-    "build": "craco build",
-    "test": "craco test --coverage --testResultsProcessor=jest-sonar-reporter",
-    "test:cov": "craco test --coverage",
+    "start": "vite",
+    "build": "vite build",
+    "test": "jest --coverage --testResultsProcessor=jest-sonar-reporter",
+    "test:cov": "jest --coverage",
     "up": "yarn && yarn start"
```

Could you add a "prepare" script here? This is a special lifecycle script that will run when `yarn install` is run for the first time, ensuring the USWDS files are created:
```suggestion
    "prepare": "gulp init",
    "start": "vite",
    "build": "vite build",
    "test": "jest --coverage --testResultsProcessor=jest-sonar-reporter",
    "test:cov": "jest --coverage",
    "up": "yarn && yarn start"
```